pkg/proto/connectionserver/api.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+package datadog.test.v1;
+
+option go_package = "/connectionserver"; // golang
+
+message Connection {
+  bytes data = 1;
```

Comment:
A doc here would be valuable (i.e. what type is this?)

Reply:
I utilized the 'bytes' data type to transmit marshalled connections. I chose this approach to avoid dealing with potential changes in the connection type. Instead, I tried to work solely with the binary representation.
Do you agree with this approach? BTW - I have added also doc in the proto file